frontend/src/components/BCDataGrid/BCDataGridServer.jsx

```diff
@@ -74,6 +75,8 @@ const BCDataGridServer = ({
   suppressPagination,
   ...others
 }) => {
+  const { t } = useTranslation(['report'])
```

Comment:
I see translation imported here but I can't see it used anywhere, did I miss it?

Reply:
Used to retrieve the constant value "complianceReport": "Compliance report":
```
localFilteredData = localFilteredData.filter((row) => {
        const typeLiteral = t('report:complianceReport').toLowerCase()
        return typeLiteral.includes(filterText)
      })
```

Reply:
Ah thanks! Just could not see it and CTRL+F with t is not a great option.